ecosystem/python/sdk/aptos_sdk/authenticator.py

```diff
@@ -109,14 +175,77 @@ def serialize(self, serializer: Serializer):
         serializer.struct(self.signature)
 
 
+class FeePayerAuthenticator:
+    sender: AccountAuthenticator
+    secondary_signers: List[typing.Tuple[AccountAddress, AccountAuthenticator]]
+    fee_payer: typing.Tuple[AccountAddress, AccountAuthenticator]
+
+    def __init__(
+        self,
+        sender: AccountAuthenticator,
+        secondary_signers: List[typing.Tuple[AccountAddress, AccountAuthenticator]],
+        fee_payer: typing.Tuple[AccountAddress, AccountAuthenticator],
```

Comment:
Tbh, this kind of tuple would have probably been easier as an object for naming purposes and readability.  Otherwise as a user you have to keep read the code later to figure out which is which.

Reply:
Hmm... maybe, not sure what we'd call it. Should look at what you guys do in the TS version. I suspect you probably follow the Rust implementation and just store more fields. Not sure that's much better. Typing is a huge win here if you use it right.